client/src/App.jsx

```diff
@@ -1,11 +1,13 @@
 import React from 'react';
 import { BrowserRouter, Route, Switch } from 'react-router-dom';
+import Login from './components/Login';
 import './App.css';
 
 export default () => (
   <BrowserRouter>
     <Switch>
       <Route exact path="/" component={() => <div />} />
+      <Route exact path="/login" component={() => <Login />} />
```

Comment:
u can use the component name directly
```suggestion
      <Route exact path="/login" component={Login} />
```